utils/api/contracts/customScript.ts

```diff
@@ -0,0 +1,22 @@
+import { initContract } from '@ts-rest/core';
+import { z } from 'zod';
+import { extendZodWithOpenApi } from '@anatine/zod-openapi';
+import { createCustomScriptSchema, customScriptSchema } from '../schemas/customScript';
+
+extendZodWithOpenApi(z);
+
+const c = initContract();
+
+export const customScriptContract = c.router({
+	set: {
+		path: '/api/customScripts',
+		method: 'POST',
+		summary: 'Set a custom script',
+		description:
+			'Set a custom scripts, e.g. the CSS or JS (if you have access) for this community',
+		body: createCustomScriptSchema,
+		responses: {
+			200: customScriptSchema,
+		},
+	},
+});
```

Comment:
This allows users to update scripts, for most only the CSS.
In the client, I've only explicitly added a method to update the CSS. People who have JS scripts activated can probably figure out how to use the raw client (e.g. `PubPub.client.customScript.set`, instead of `PubPub.community.css`)